ruby/vampires.rb

```diff
@@ -0,0 +1,110 @@
+def age_check(age, year)
+	if 2017 - age == year
+		return true
+	elsif 2016 - age == year
+		return true
+	else
+		return false
+	end;
+end;
+
+def wants_garlic_bread(input)
+	case input
+	when "yes", "yea", "ya", "yep", "ye"
+		return true
+	when "no", "nah", "nay", "negative", "nope"
+		return false
+	else  
+		return false
+	end;
+end;
+
+def wants_insurance(input)
+	case input
+	when "yes", "yea", "ya", "yep", "ye"
+		return true
+	when "no", "nah", "nay", "negative", "nope"
+		return false
+	else
+		return false
+	end;
+end;
+
+def ask_questions 
+	puts "How many employees are you processing?"
+	number_of_employees = gets.chomp.to_i

```

Comment:
When using methods we try to avoid using `get.chomp` by design.

Instead find a way to refactor this to pass a parameter in and keep `gets.chomp` in your driver code.